extensions/amp-story-interactive/amp-story-interactive.md

```diff
@@ -1,10 +1,315 @@
+
+Complete URL of backend. Stores interactive element votes.
+
+### theme (optional)
+
+Controls the color of the chips and text. Can be `light` (default), `dark`.
+
+### chip-style (optional for poll, quiz, results)
+
+Controls the style of the component. Can be `flat` (default), `shadow`, or `transparent`.
+Results and binary-poll elements don't support shadow.
+
+### prompt-text (optional)
+
+Adds a prompt to the top of the component. Use `prompt-text` to write the poll/quiz question, or as a prefix to the category in the `amp-story-interactive-result` element.
+
+### prompt-size (optional for binary-poll, poll, quiz)
+
+Controls the `font-size` of the prompt text, so it only applies for elements that have prompts. Can be `small` (), `medium` (default), `large`. Large prompts will hold up to 3 lines of text, other sizes will hold up to 4 lines of text.
+
+### option-{1/2/3/4}-text (required)
+
+String that represents a numbered option. Binary polls require 2 options. Polls and quizzes may include between 2 and 4 options.
+
+The `amp-story-interactive-result` element uses this string value as category description.
+
+### option-{1/2/3/4}-confetti (optional for binary-poll, poll, quiz)
+
+Emoji that bursts in an explosion animation when an option is selected. On quizzes, only the correct option should have a confetti.
+
+### option-{1/2/3/4}-results-category (optional for poll, required for results)
+
+On the results component, this attribute represents the name of the category, shown in larger text after the `prompt-text` and before the category description. If the results element doesn't specify thresholds, the category strategy will be used: the category with more options selected in polls across the story will be shown.
+
+On polls it links the options to the result with that name as mentioned above. The string has to match perfectly for the options to be linked.
+
+### option-{1/2/3/4}-results-threshold (optional for results)
+
+On the results element, it determines the lower boundary for the category when linked to quizzes. The component will calculate the score as a percentage of questions answered correctly (between 0 and 100), and it will show the category that has the best lower threshold that matches the score. The best threshold is the highest one that is lower or equal to the score, or the lowest score if all thresholds are higher than the score. If a threshold is present for any option, all other options also need a threshold.
+
+## Aggregate data source
+
+All selectable interactive elements (binary-poll, poll, quiz) show the percentage of users that selected each option. The backend specified with the `endpoint` attribute will store the aggregate data for the interaction following the API described below.
+To fetch the data for an interactive element, the necessary fields are:
+
+- <div id="interactiveId"></div>`interactiveId`: the `base64encode(CANONICAL_URL) + "+" + element.id`
```

Comment:
something is weird with the formatting here - maybe place a space between the closing div tag and the backtick?

Reply:
Sure, I want to link another part of this doc to here and seems to work. Will add a space